Makefile

```diff
@@ -237,10 +254,19 @@ VERIFY_TAGS = 0
 .PHONY: $(DOCKER_PUSH_TARGETS)
 $(DOCKER_PUSH_TARGETS): NAME=$(@:docker-push-%=%)
 $(DOCKER_PUSH_TARGETS):
-	@if [ $(VERIFY_TAGS) != 1 ]; then make docker-build-$(NAME) && docker push "$(CTR_REGISTRY)/$(NAME):$(CTR_TAG)"; else bash scripts/publish-image.sh $(NAME); fi
+	@if [ $(VERIFY_TAGS) != 1 ]; then make docker-build-$(NAME) && docker push "$(CTR_REGISTRY)/$(NAME):$(CTR_TAG)"; else bash scripts/publish-image.sh $(NAME) "linux"; fi
+
+
+DOCKER_PUSH_WINDOWS_TARGETS = $(addprefix docker-push-windows-, $(DEMO_TARGETS))
+VERIFY_TAGS = 0
+.PHONY: $(DOCKER_PUSH_WINDOWS_TARGETS)
+$(DOCKER_PUSH_WINDOWS_TARGETS): NAME=$(@:docker-push-%=%)
+$(DOCKER_PUSH_WINDOWS_TARGETS):
+	@if [ $(VERIFY_TAGS) != 1 ]; then make ARGS=--push docker-build-$(NAME); else bash scripts/publish-image.sh $(addprefix windows-, $(NAME)) "windows"; fi
+
 
 .PHONY: docker-push
-docker-push: $(DOCKER_PUSH_TARGETS)
+docker-push: $(DOCKER_PUSH_TARGETS) $(DOCKER_PUSH_WINDOWS_TARGETS)
```

Comment:
The demo script does invoke `make docker-push`, which already does more than it really needs to. Let's at least make it easy to build all the images for only windows or linux so we can use that in the demo script. Then we can have an all-in-one target we call from CI.